CMakeLists.txt

```diff
@@ -576,9 +621,9 @@ set(LIB_DESTINATION "lib${LIB_SUFFIX}")
 
 # library
 
+
 add_library(audaspace ${LIBRARY_TYPE} ${SRC} ${HDR})
-target_link_libraries(audaspace ${LIBRARIES})
-set_target_properties(audaspace PROPERTIES SOVERSION ${AUDASPACE_VERSION})
```

Comment:
You add FFTW_LIBRARY to LIBRARIES, this should be enough for ${LIBRARIES} to include the fftw library, no? Also please keep the SOVERSION property.